prebuilds.mk

```diff
@@ -41,11 +41,11 @@ DKILLCONV_PACKAGE=$(notdir $(DKILLCONV_DOWNLOAD))
 
 # Tools and libraries to be used for the target system
 # Currently, the target is always windows-mingw32
-SDL_DOWNLOAD=https://www.libsdl.org/release/SDL-devel-1.2.15-mingw32.tar.gz
+SDL_DOWNLOAD=http://www.libsdl.org/release/SDL2-devel-2.0.12-mingw.tar.gz
```

Comment:
https?

Reply:
Links from official SDL page http://libsdl.org/download-2.0.php.
Anyway we're just downloading file and not sending any sensible info to the web so protocol does not really matter.

Reply:
Someone can give you fake sources and you can compile them into your project, but yes that is not important here

Reply:
Yep, if someone takes over your DNS, they can give you anything :D